### ISSUE TITLE
[BUG] Literal \n in crc.log

### ISSUE THREAD

**Issue by syedriko** [code-ready/crc#698]
### General information

  * OS: Linux, Fedora 30
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
$ ./crc version
version: 1.0.0-rc.0+34371d3
OpenShift version: 4.2.0-0.nightly-2019-09-26-192831 (embedded in binary)
$ 
```
  
## CRC status
```bash
$ ./crc status
ERRO Error occurred: Machine "crc" does not exist. Use "crc start" to add a new one. 
$
```

## CRC config
```bash
$ ./crc config view
$
```

## Host Operating System
```bash
$ cat /etc/os-release
NAME=Fedora
VERSION="30 (Workstation Edition)"
ID=fedora
VERSION_ID=30
VERSION_CODENAME=""
PLATFORM_ID="platform:f30"
PRETTY_NAME="Fedora 30 (Workstation Edition)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:30"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=30
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=30
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
$
```

### Steps to reproduce

  1. crc setup
  2. grep '\\n' ~/.crc/crc.log

### Expected
No output from "grep '\\n' ~/.crc/crc.log"

### Actual
$ grep '\\n' ~/.crc/crc.log
time="2019-10-08T16:31:17-04:00" level=debug msg="Download saved to /tmp/crc316183009/oc.tar.gz \n"
time="2019-10-08T16:31:18-04:00" level=debug msg="Copying '/tmp/crc316183009/oc' to '/home/syedriko/.crc/bin/oc'\n"
time="2019-10-08T16:31:18-04:00" level=debug msg="Checking if libvirt version is >=3.4.0\n"
time="2019-10-08T16:31:19-04:00" level=debug msg="Download saved to /home/syedriko/.crc/bin/crc-driver-libvirt \n"
time="2019-10-08T20:57:58-04:00" level=debug msg="Download saved to /tmp/crc285030223/oc.tar.gz \n"
$ 

### Logs
Here's the complete log from a "crc setup": https://gist.github.com/syedriko/d1a87aec52a27cbe74132440c3417a2b

**Comment by praveenkumar** [comment]
Fixed by #699

**Comment by gbraad** [comment]
Built on Travis failed due to time-out. Ignore ...
